pkg/sql/executor_ir.go

```diff
@@ -93,6 +94,33 @@ func RunSQLProgram(sqlProgram string, db *DB, modelDir string, session *pb.Sessi
 	return rd
 }
 
+// ParseSQLStatement parse the input SQL statement and output IR in probobuf format
+func ParseSQLStatement(sqlProgram string, session *pb.Session) (string, error) {
+	connStr := session.DbConnStr
+	driverName := strings.Split(connStr, "://")[0]
+	sqls, err := parse(driverName, sqlProgram)
```

Comment:
If we have renamed `sqlProgram` to `sql`, maybe change calling `parse` to calling `parseOneStatemeent`.